src/plugins/intel_gna/CMakeLists.txt

```diff
@@ -71,20 +71,39 @@ set_target_properties(${TARGET_NAME} ${TARGET_NAME}_test_static
 
 # install
 
-file(GLOB_RECURSE source_list "${libGNA_LIBRARIES_BASE_PATH}/*${CMAKE_SHARED_LIBRARY_SUFFIX}*")
-install(FILES ${source_list}
-        DESTINATION ${IE_CPACK_RUNTIME_PATH}
-        COMPONENT gna)
+file(GLOB_RECURSE gna_libraries "${libGNA_LIBRARIES_BASE_PATH}/*${CMAKE_SHARED_LIBRARY_SUFFIX}*")
+set(gna_library_installed OFF)
+
+foreach(gna_lib IN LISTS gna_libraries)
+    # install only versioned files like libgna.so.N or libgna.so.N.M
+    # and don't install global symlink libgna.so. Exceptions:
+    # 1. On Windows we don't have libraries versioning and need to install all .dlls
+    # 2. For case of static libraries we also need to install .so since it's required for linkage with final application
+    if(gna_lib MATCHES "^.*${CMAKE_SHARED_LIBRARY_SUFFIX}\(\.[0-9]+\)+$" OR
```

Comment:
It would be great to explain the regular expression in detail for future developers.

Reply:
documentation for cmake explains regular expressions :)

Reply:
I think Anton meant that we need to explain what we are filtering here by the expression, but the current description is good enough)